src/lbaf/Execution/lbsBruteForceAlgorithm.py

```diff
@@ -63,12 +63,12 @@ def compute_arrangement_works(self, objects: tuple, arrangement: tuple) -> dict:
         # Return arrangement works
         return works
 
-    def execute(self, phase: Phase, distributions: dict, statistics: dict, _):
+    def execute(self, phases: list, distributions: dict, statistics: dict, _):
         """ Execute brute force optimization algorithm on Phase instance
         """
-
-        # Ensure that a phase was properly passed
-        if not isinstance(phase, Phase):
+        # Ensure that a list with at least one phase was provided
+        if not phases or not isinstance(phases, list) or not isinstance(
+            (phase := phases[0]), Phase):
```

Comment:
should be in line 70. Then line 70 will have 102 chars < 120, so it's ok, more readable

Reply:
I like it better like it is now

Reply:
Then it is again the same problem. Indentation is wrong, why? 
PEP 8: E125 continuation line with same indent as next logical line
Because the indentation should be different than lines inside if:
WRONG:
```Python3
if not phases or not isinstance(phases, list) or not isinstance(
    (phase := phases[0]), Phase):
    self.__logger.error(f"Algorithm execution requires a Phase instance")
    raise SystemExit(1)
self.phase = phase
```

CORRECT:
```Python3
if not phases or not isinstance(phases, list) or not isinstance(
        (phase := phases[0]), Phase):
    self.__logger.error(f"Algorithm execution requires a Phase instance")
    raise SystemExit(1)
self.phase = phase
```

Reply:
I don't know but that's a detail; please let's not waste our time with small things like this
We can always do a global style sweep later
We have bigger fish to fry now